index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome To Ayodeji's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <section class="header_headline_section">
+        <header>
+        <nav>
+            <div class="header_logo"><a href="" class="logo">My Logo</a></div>
+            <div class="header_icon"><img src="icons/Icon-Menu.png" alt="" srcset=""></div>
+        </nav>
+    </header>
+    <section class="headline">
+        <p class="intro_text">I’m Ayodeji. Glad to see you!</p>
```

Comment:
- [ ] The main title ("I'm Ayodeji ...") is a `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag.